common/server.conf

```diff
@@ -8,3 +8,17 @@ ssl_ciphers EECDH+AES128:RSA+AES128:EECDH+AES256:RSA+AES256:EECDH+3DES:RSA+3DES:
 ssl_prefer_server_ciphers on;
 
 server_name dcos.*;
+
+location /system/logs/v1/ {
+    access_by_lua 'auth.validate_jwt_or_exit()';
+    proxy_buffering off;
+    proxy_cache off;
+
+    proxy_set_header Connection '';
```

Comment:
Could you please add some comment here regarding why we want to keep upstream connection up after request ? We can just copy the text from the PR description and i.e. link this SO for more details: http://stackoverflow.com/questions/13672743/eventsource-server-sent-events-through-nginx.